sources/tools/Stride.Importer.3D/MeshConverter.cs

```diff
@@ -547,14 +793,33 @@ private unsafe void RegisterNodes(Node* fromNode, int parentIndex, Dictionary<In
 
             nodes.Add(modelNodeDefinition);
 
+            if (nodePointerToNodeIndex is not null)
+            {
+                nodePointerToNodeIndex.Add((IntPtr)fromNode, nodeIndex);
+            }
+            if (duplicateNodeNameToNodeIndices is not null)
+            {
+                string originalNodeName = fromNode->MName.AsString.CleanNodeName();
+                if (!string.IsNullOrWhiteSpace(originalNodeName) && !string.Equals(originalNodeName, modelNodeDefinition.Name))
+                {
+                    if (!duplicateNodeNameToNodeIndices.TryGetValue(originalNodeName, out var nodeIndices))
+                    {
+                        nodeIndices = new();
+                        duplicateNodeNameToNodeIndices[originalNodeName] = nodeIndices;
+                        //Logger.Warning($"Duplicate node name found: {originalNodeName}. Animations may not play correctly if this node is used in an animation.");
```

Comment:
Unrelated commented out line afaict ?

Reply:
I was hesitant on whether to log a warning here, because technically the whole point of the other code is to actually try to resolve the duplicate nodes.
But now I just realized I can just output this at Info level.

Reply:
Done. Changed to output at Info level, and rebased the commit to the latest.